ui-core/src/types/forms.ts

```diff
@@ -227,6 +227,11 @@ export type HtmlQuestion = BaseQuestion & {
   html: I18nSurveyElement
 }
 
+export type PanelDynamicQuestion = TitledQuestion & {
```

Comment:
is this used anywhere?

Reply:
good catch - artifact from experimenting with a different approach